File: go/tarifa.go
package main

import (
	"fmt"
)

func main() {

	var x int
	fmt.Scan(&x)

	var n int
	fmt.Scan(&n)

	var leftover int

	for i := 0; i < n; i++ {
		var spent int
		fmt.Scan(&spent)

		leftover += x - spent

	}

	fmt.Println(leftover + x)

}

File: c/tarifa.c
#include <stdio.h>
#include <string.h>

int main() {
	int x, n;
	(void) scanf("%d %d", &x, &n);
	
	int leftover = 0;
	for (int i = 0; i < n; i++) {
		int spent;
		scanf("%d", &spent);

		leftover += x - spent;
	}

	printf("%d\n", leftover + x);
	
}
